service/internal/integrationtest/s3control/setup_test.go

```diff
@@ -0,0 +1,121 @@
+// +build integration
+
+package s3control
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/internal/integrationtest"
+	"github.com/aws/aws-sdk-go-v2/service/s3control"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+)
+
+var (
+	svc                            *s3control.Client
+	s3ControlEndpoint, stsEndpoint string
+	accountID                      string
+	insecureTLS, useDualstack      bool
+)
+
+var region = "us-east-1"
```

Comment:
Does this still need to default to `us-east-1` can it use `us-west-2`. thats what most of the other integration tests default to.